### ISSUE TITLE
FMD clean install FMD completely and utterly broken nothing works

### ISSUE THREAD

**Issue by molitar** [fmd-project-team/FMD#943]
Ok to see if I could get Kissmanga to work I did a complete brand new install just in case an old configuration was causing the problem.

Now I am unable to download any Data from the Manga Server so I can not even test.  I just get 500 Internal Server Error and 503: Backend Unhealthy.

Nothing is working on a brand new install at all!

**Comment by 7h3Cub3** [comment]
Check another manga site, because kissmanga is well known to be extremely unreliable and doesn't work 70% of the time with FMD, try mangadex for example to check if everything is correct iwht your installation. The return message you get are obviously and issue with kissmanga not FMD....

**Comment by molitar** [comment]
It started working but not until later in the day. Maybe the server itself was down.

**Comment by Tmp341** [comment]
Duplicate of #934
